xbmc/video/VideoInfoTag.cpp

```diff
@@ -221,12 +221,12 @@ bool CVideoInfoTag::Save(TiXmlNode *node, const std::string &tag, bool savePathI
   }
   XMLUtils::SetStringArray(movie, "genre", m_genre);
   XMLUtils::SetStringArray(movie, "country", m_country);
-  if (!m_strSet.empty())
+  if (!m_set.title.empty())
   {
     TiXmlElement set("set");
-    XMLUtils::SetString(&set, "name", m_strSet);
-    if (m_strSetOverview.empty())
-      XMLUtils::SetString(&set, "overview", m_strSetOverview);
+    XMLUtils::SetString(&set, "name", m_set.title);
+    if (m_set.overview.empty())
+XMLUtils::SetString(&set, "overview", m_set.overview);
```

Comment:
indentation is wrong

Reply:
indeed. will wait for build to finish before pushing fix.